### PR TITLE
DEV-469 Create NPM packages for eslint and prettier configs

### PR DESCRIPTION
**Pull request by dhjonesclc**
# Overview
Update this project to publish to npm as a private scoped package when merged to master.

# Changes
- Update `package.json` to scope package name 
- Add semantic release for auto publishing

DEV-469

### CODE REVIEW

**Review by oddirmeyer — APPROVED** [review]
Visual review ✅

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,9 +1,32 @@
...
+        
+  publish:
+    <<: *defaults
     steps:
       - checkout
-      - run: echo "simple build executed"
\ No newline at end of file
+      - run:
+          name: Set NPMJS Registry
+          command: npm config set //registry.npmjs.org/:_authToken ${NPM_TOKEN}
```
Comment (oddirmeyer) [inline .circleci/config.yml:16]: Before we merge, we should add the CircleCI project to specify this environment variable and another Github token and deploy key.

Comment (dhjonesclc) [reply .circleci/config.yml:16]: Yep, got those set up for this project and the eslint-config-clc project as well,.

**Review by zsidnam — APPROVED** [review]
Awesome, thanks for doing this! 👍

**Pull request merged**